nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -169,6 +174,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
+        session.getProvenanceReporter().send(flowFile, flowFile.getAttribute(HTTP_REQUEST_URI), stopWatch.getElapsed(TimeUnit.MILLISECONDS));
```

Comment:
In this case, we are performing a SEND event, so it should be the information that we are sending the data to (the client). Looking at this a bit more, I think the URI used for the HandleHttpRequest is also wrong. They both should be indicating the client's hostname, as this is where the data was Received from and Sent to. Right now, I believe, the HandleHttpRequest is using just the path of the URL (the Request URI), and that is a bug.